### ISSUE TITLE
Offset isn't recognized on first click

### ISSUE THREAD

**Issue by drezannis** [flesler/jquery.localScroll#31]
Everything works as expected except when I click on one of the localScroll enabled links, the offset is not respected (the page is scrolled to the top of the containing div instead of 'top of the containing div + offest'). If I click on the localScroll enabled link a second time (or any time thereafter, or any other link) the page is scrolled in accordance with the specified offset.

I'm at a loss. Any help is greatly appreciated. 

`$(document).ready(function(){                     
  $('.local-scroll-enabled').localScroll({
    offset: {
      top: -60
    }
  });
})`

**Comment by flesler** [comment]
Hi, got a demo page?

**Comment by drezannis** [comment]
Thank you for the prompt response. I ended up altering the layout to remove the issue. 

For anyone facing this same problem, it was caused by a sticky nav's height not being factored into the offset because it was not yet sticky (it was centered vertically on page load and would only become sticky once it hit the top of the window). 

So on a 2nd click, the stick nav was sticky and correctly factored into the offset (I don't think I'm really wording that correctly).

Basically localScroll was working as intended, it was a sticky item changing where localScroll was calculating what the div should be offset from. 

Hope that makes sense and helps someone else!

Ariel thanks for an amazing plugin!
